### ISSUE TITLE
Placeholder

### ISSUE THREAD

**Issue by ghost** [clyall/FormEmailer#13]
Howdy!
How can I insert a placeholder on the "to" field that allows me to auto-mail someone who fills in my form, without manually inputting that address?

**Comment by jac68** [comment]
The user will input his/her email in the form.
Just type that question's placeholder in the "to" field (FormEmailset setup window) and setup trigger.

**Comment by ghost** [comment]
Hi jac68!
I have the "to" field in my form. But when configuring the FormEmailer, what do I fill in the "to" field with? I got everything working fine, except for that small detail.

**Comment by jac68** [comment]
You will have a "to" column in your responses sheet.

If you enter #to# in the FormEmailer settings window (and don't forget to save&close and setup trigger) it should work.

-------

I am sure you already have done that, so maybe your problem is that you added the "to" question after installing the script.

If that is the case I suggest you start all over again: link your form to a new sheet and install script.

Probably there is a faster solution, but I found similar issues and instead of messing with the FormEmailer setup tab, I just solved that way.

Hope it helps.
